packages/x-components/src/x-installer/api/api.types.ts

```diff
@@ -67,6 +78,8 @@ export interface SnippetConfig {
   documentDirection?: DocumentDirection;
   /** The currency name. There should be a currency format associated to this name in the app. */
   currency?: string;
+  /** Callbacks to be triggered when an XEvent is emitted. */
+  callbacks?: XEventListeners;
```

Comment:
I think this type has to be a `Partial`:

![image](https://user-images.githubusercontent.com/3502075/143579784-b6eb0e6c-15f5-43dc-81da-14b28beacead.png)